python/ray/tune/scripts.py

```diff
@@ -35,10 +42,17 @@ def list_trials(experiment_path, sort, output):
     "-o",
     default=None,
     type=str,
-    help="Select filename to output information to.")
-def list_experiments(project_path, sort, output):
+    help="Select file to output information to.")
+@click.option(
+    "--filter",
+    "--filter_op",
```

Comment:
I think these might need to be flipped. Otherwise you get an error:
```
$ tune list-experiments ~/ray_results/gym/Point2DEnv/Default-v0 --filter "total_trials > 1"
Traceback (most recent call last):
  File "/home/kristian/conda/envs/softlearning/bin/tune", line 11, in <module>
    load_entry_point('ray', 'console_scripts', 'tune')()
  File "/home/kristian/conda/envs/softlearning/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/kristian/conda/envs/softlearning/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/kristian/conda/envs/softlearning/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/kristian/conda/envs/softlearning/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/kristian/conda/envs/softlearning/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
TypeError: list_experiments() got an unexpected keyword argument 'filter'
```

Reply:
I just pushed change for these.

Reply:
Hmm that's really weird. I originally had "filter_op" first for exactly that reason. My understanding was that the first flag must match the argument name passed into the function. After pulling your changes and running the same command, I get the same error:

```
$ python scripts.py list-experiments ~/ray_results/my_project/ --filter "total_trials > 1"
Traceback (most recent call last):
  File "scripts.py", line 82, in <module>
    main()
  File "scripts.py", line 78, in main
    return cli()
  File "/Users/andrewtan/anaconda/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/andrewtan/anaconda/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/andrewtan/anaconda/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/andrewtan/anaconda/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/andrewtan/anaconda/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
TypeError: list_experiments() got an unexpected keyword argument 'filter'
```

I'm not sure why we're getting different results for the same command. I'm on version style-1.1.0 update-0.0.1 for click.

Reply:
Intersting. I'm on click 7.0, so maybe there's been some recent change in the api. Looking at the documentation, seems like we can fix this by using non-dashed names like:
```python
@click.option(
    "--filter",
    "filter_op",
    ...)
```

This seems to work on my end. Can you verify that this on your end too?

Reply:
Ah okay yeah I was running 6.7 previously. Upgraded and ran the new code--looks good on my end!